python/ray/dataframe/dataframe.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 from pandas.api.types import is_scalar
 from pandas.util._validators import validate_bool_kwarg
-from pandas.core.index import _ensure_index_from_sequences
+# from pandas.core.index import _ensure_index_from_sequences
```

Comment:
why did this get commented out? If we're no longer using it then let's just remove it.